src/dist/replication/meta_server/meta_server_failure_detector.cpp

```diff
@@ -263,6 +263,10 @@ void meta_server_failure_detector::on_ping(const fd::beacon_msg& beacon, ::dsn::
     {
         ack.is_master = false;
         ack.primary_node = get_primary();
+        {
```

Comment:
For example: 
1. There is an alive meta server M, but it is not master.
2. There is an replica server R.

R send beacon to M. Although M is not master, R will regard itself alive as the RPC response is ERR_OK. If M becomes the master before the second beacon from R arrives, the perfect FD rule will be broken.